source/conf.py

```diff
@@ -195,7 +195,7 @@ def setup(app):
 #html_domain_indices = True
 
 # If false, no index is generated.
-#html_use_index = True
+html_use_index = False
```

Comment:
Does this change need to be tested locally? If you haven't done so already? I'm not clear what this is doing.